src/main/java/com/gluonhq/substrate/target/LinuxTargetConfiguration.java

```diff
@@ -122,6 +123,22 @@ public boolean link() throws IOException, InterruptedException {
         return linkFlags;
     }
 
+    @Override
+    List<String> getTargetSpecificAOTCompileFlags() throws IOException {
+        if (!crossCompile) {
+            return super.getTargetSpecificAOTCompileFlags();
+        }
+        Path llcPath = getLlcPath();
+        return Arrays.asList("-H:CompilerBackend=" + Constants.BACKEND_LLVM,
+                "-H:-SpawnIsolates",
+                "-Dsvm.targetArch=" + projectConfiguration.getTargetTriplet().getArch(),
+                "-H:+UseOnlyWritableBootImageHeap",
+                "-H:+UseCAPCache",
+                "-H:CAPCacheDir=/tmp/cca",
```

Comment:
Is this a hardcoded path? Will native-image create it if it doesn't exist? Should we use a system property or `Files.createTempDirectory("cca")`?